docs/reference/edot-sdks/android/index.md

```diff
@@ -0,0 +1,78 @@
+- The last span is also created within your backend service when it queries other resources internally. In this case, it makes an HTTP request to another service, but in other cases, it could involve querying a database.
+
+You can select any of the spans to view their full details if you need more contextual information from each.
+
+:::{note}
+For distributed tracing to work properly, configure your backend services to send telemetry to the {{stack}} as well.
+:::
+
+## Session review
+
+The SDK attaches session information to each span and log generated from your application. This allows you to create queries that group all the telemetry that belongs to a session and form a session event timeline. This is useful to identify the most common actions performed by your users, as well as tracing the steps leading up to errors they might encounter.
+
+## Alerts and dashboards
+
+Because the SDK also provides [direct access](manual-instrumentation.md) to the [OpenTelemetry SDK](https://opentelemetry.io/docs/languages/java/), you can generate your own data in ways that best suit your needs and take advantage of {{stack}}'s tools, such as:
+
+ * [Create alerts](docs-content://explore-analyze/alerts-cases.md) when something interesting happens; for example, when an error is recorded.
+ * [Build custom dashboards](docs-content://explore-analyze/dashboards.md) to display your data the way you need to see it
+
+## Features
+
+The EDOT Android SDK provides additional features on top of those that come with the [OpenTelemetry SDK](https://opentelemetry.io/docs/languages/java/).
+
+### Disk buffering
+
+Your application's telemetry data is stored locally before being sent to the {{stack}}. Then it's removed either after being successfully exported or to make room for new telemetry data if needed. This minimizes the risk of data loss due to internet connectivity issues.
+
+### Real time
+
+For [distributed tracing](#distributed-tracing) to work properly, your application's time should be in sync with the [coordinated universal time](https://en.wikipedia.org/wiki/Coordinated_Universal_Time). This is sometimes an issue for Android applications, as the time provided by the OS is often not accurate enough. The SDK aims to synchronize telemetry timestamps with the universal time to ensure a reliable view of event timelines.
+
+### Sessions
+
+A session groups telemetry data within a time frame when your application is active by adding a `session.id` attribute to all the spans and logs that come out of your application. This helps limit the volume of data you need to look at when investigating the steps that led to an error within your app. It can also give you insight into the common actions performed on each use.
+
+A session is created when no previous session exists or when the previous one has expired. Sessions expire after 30 minutes of inactivity. You can extend this for up to 4 hours.
```

Comment:
```suggestion
A session is created when no previous session exists or when the previous one has expired. Sessions expire after 30 minutes of inactivity. This will be extended to up to 4 hours when there's no inactivity.
```

Reply:
Users can't currently manually extend it, as it's suggested in the text. It will get extended automatically when there's always telemetry being generated without an idle time of 30 min.